jruby/plan.sh

```diff
@@ -7,15 +7,15 @@ pkg_upstream_url=https://github.com/jruby/jruby
 pkg_source="https://github.com/jruby/jruby/archive/${pkg_version}.tar.gz"
 pkg_shasum=9b926cbca56faa9d724753c044c89d86bb48c0b5c26c5bf39120e89dc2fb0c69
 pkg_license=('EPL 1.0, GPL 2 and LGPL 2.1')
-pkg_deps=(core/glibc core/jre8 core/bash core/coreutils)
-pkg_build_deps=(core/which core/make core/jdk8)
+pkg_deps=(core/glibc core/openjdk11 core/bash core/coreutils)
+pkg_build_deps=(core/which core/make core/openjdk11)
```

Comment:
This shouldn't be necessary: #https://github.com/habitat-sh/core-plans/pull/2875/files#r306937636 for additional context.

I also think we should use corretto11 in this case,  since we're using coretto8 for jruby1.